src/Cooker/providers/GitProvider/getLatestTagMatchingName.js

```diff
@@ -21,10 +21,13 @@ export function getLatestTagMatchingName(repoPath, regex) {
           )
         )
       )
-      .then(tags =>
-        tags
-          .sort((a, b) => (a.date > b.date ? -1 : 1))
+      .then(tags => {
+        return tags
+          .sort(
+            (a, b) =>
+              new Date(a.date).getTime() > new Date(b.date).getTime() ? -1 : 1
```

Comment:
Why convert back to a date ? Doesn't the ISO date sort correctly as a string: "2017-07-16T16:25:05.158Z" and no need for { return } since we are having a single statement...